zad5/app/views/artists/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="container">
+  <%= link_to 'Add new artist', new_artist_url, class: 'btn btn-success pull-right' %>
+  <h2>Artist Table</h2>
+  <table class="table">
+    <thead>
+      <tr>
+        <th class="text-left">Artists</th>
+        <th class="text-right">Actions</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @artists.each do |artist| %>
+        <tr>
+          <td><%= artist.name %></td>
+          <td class="text-right">
+            <%= link_to 'edit', edit_artist_url(artist), class: 'btn btn-warning' %>
+            <%= link_to 'remove', artist_url(artist), class: 'btn btn-danger', method: :delete %>
```

Comment:
można dodać `data: { confirm: 'Are you sure?' }`, aby przypadkiem nie usunąć niczego 🙈